components/brave_today/browser/direct_feed_controller.cc

```diff
@@ -207,15 +190,16 @@ void DirectFeedController::FindFeeds(
             auto* loader = iter->get();
             auto response_code = -1;
             std::string mime_type;
-            std::string charset = GetResponseCharset(loader);
             GURL final_url(loader->GetFinalURL());
             base::flat_map<std::string, std::string> headers;
+            std::string charset = "utf8";
             if (loader->ResponseInfo()) {
               auto headers_list = loader->ResponseInfo()->headers;
               mime_type = loader->ResponseInfo()->mime_type;
               if (headers_list) {
                 response_code = headers_list->response_code();
               }
+              charset = loader->ResponseInfo()->charset;
```

Comment:
`GetResponseCharset` substituted `utf-8` for an empty value from the header. Is that not still important?